src/ray/common/ray_config_def.h

```diff
@@ -480,7 +480,7 @@ RAY_CONFIG(uint64_t, task_events_send_batch_size, 10 * 1000)
 /// report gRPC call. A task could have more profile events in GCS from multiple
 /// report gRPC call.
 /// Setting the value to -1 allows unlimited profile events to be sent.
-RAY_CONFIG(int64_t, task_events_max_num_profile_events_for_task, 100)
+RAY_CONFIG(int64_t, task_events_max_num_profile_events_for_task, 1000)
```

Comment:
Let's just keep it 100? 100 looks big enough actually.

Reply:
So this happens when a driver keeps submitting tasks, since driver task id is fixed for the lifetime of a driver (driver is just running one fake driver task), all profile events generated by the driver will have the same taskid + attempt number, and this could create a HUGE rpc::TaskEvents with thousands of profile events.